tests/test_compute_funs.py

```diff
@@ -1189,6 +1189,20 @@ def test_compare_quantities_to_vmec():
     np.testing.assert_allclose(J_dot_B_desc, J_dot_B_vmec, rtol=0.005)
 
 
+do_not_test_these_values = {
+    "desc.geometry.curve.SplineXYZCurve": [
+        "x_s",
```

Comment:
why not test these? It's a regression test so as long as we're comparing apples to apples it should be ok? We can have a separate test that compares things to spectral basis curves

Reply:
Thought something different was happening with the test, just regened the master data for spline XYZ curve and it works now so they are tested now